src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -607,6 +610,11 @@ private static String getNormalizedPointsListColorizedDescending(int[] subs, int
             }
             result.add(getPointsAsColorizedHtml(subs[i]));
         }
+        
+        if (result.isEmpty()) {
+            return "<span class=\"color_neutral\" data-toggle=\"tooltip\" data-placement=\"top\" "
+                    + "title=\"Not Available: There is no data for this or the data is not enough\">N/A</span>";
```

Comment:
Use the constant <code>Const.Tooltips.FEEDBACK_CONTRIBUTION_NOT_AVAILABLE</code>